webserver/__init__.py

```diff
@@ -10,6 +10,10 @@ def create_postgres():
     from postgres_connection import init_postgres_connection
     return init_postgres_connection(current_app.config['SQLALCHEMY_DATABASE_URI'])
 
+def create_redis():
```

Comment:
SQLAlchemy engine should be created once during app creation. Each set of requests is then supposed to create a connection from it.